repos/system_upgrade/el7toel8/actors/addupgradebootentry/libraries/addupgradebootentry.py

```diff
@@ -7,21 +7,24 @@
 from leapp.models import BootContent
 
 
-def add_boot_entry():
+def add_boot_entry(config=None):
     debug = 'debug' if os.getenv('LEAPP_DEBUG', '0') == '1' else ''
 
     kernel_dst_path, initram_dst_path = get_boot_file_paths()
     try:
-        _remove_old_upgrade_boot_entry(kernel_dst_path)
-        run([
+        _remove_old_upgrade_boot_entry(kernel_dst_path, config=config)
+        cmd = [
             '/usr/sbin/grubby',
             '--add-kernel', '{0}'.format(kernel_dst_path),
             '--initrd', '{0}'.format(initram_dst_path),
             '--title', 'RHEL-Upgrade-Initramfs',
             '--copy-default',
             '--make-default',
             '--args', '{DEBUG} enforcing=0 rd.plymouth=0 plymouth.enable=0'.format(DEBUG=debug)
-        ])
+        ]
+        if config:
+            cmd += ['-c'] + [config]
```

Comment:
Did you mean this?
```suggestion
            cmd += ['-c', config]
```

Reply:
@pirat89 it was already fixed by that time :)

Reply:
@pirat89 aah but yeah I see the difference now. The suggestion is probably more readable :)